src/reports.coffee

```diff
@@ -84,7 +87,35 @@ sendReports = (job, flag, done) ->
             report.type = 'Daily'
           else
             report.type = 'Weekly'
+
+          report.instance = config.alerts.himInstance
+
           try
+            i = 0
+            for data in report.data
```

Comment:
Minor: I think here you could use `for data, i in report.data` then you won't have to have your own `i` variable or increment it.